### PR TITLE
General Improvements

### PR DESCRIPTION
**Pull request by agucova**
I've made some improvements on the README, fixed some typos, updated the documentation, and (I hope you don't mind), standardized the formatting in the source code using clang-format.

(Bacán tu proyecto!)

### CODE REVIEW

**Comment by glats** [comment]
Hi! I'm grateful for your contribution!
I saw that you changed the format of the code and I wonder if you have any tool for that?

File: README.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
-# kbdlight
-
+# kbdlight 💡
 # About
-This program allows the user to change the keyboard brightness in GNU/Linux. It hasn't any extra dependency such as [upower](https://upower.freedesktop.org/) (that allow to control the brightness, but contains a lot other libs that sometimes you dont need).
-In a few words this program adapt unix's philosophy.
+This command-line utility allows the user to change the brightness of a backlit keyboard in GNU/Linux.
```
Comment (glats) [inline README.md]: Theres a typo, it says "backlit"

File: README.md [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
+
+The main advantage over alternatives like [upower](https://upower.freedesktop.org/) is the lack of unnecessary dependencies and libraries, as well as simplicity of operation. (full of UNIX goodness!)
 
 # How it works
-The program reads the file provided by the [kernel](https://www.kernel.org/) in ```/sys/class/leds```  
-It makes the basic control: increase and decrease the brightness one by one. This depends of the value in ```max_brightness``` file
+The program works in userspace by changing the value in the `brightness` file provided by the [kernel](https://www.kernel.org/) in `/sys/class/leds` (see [here](https://www.kernel.org/doc/Documentation/leds/leds-class.txt) for details).
```
Comment (glats) [inline README.md]: Cool! i was looking for that information!

File: README.md (hunk 1 above)
Comment (agucova) [reply README.md]: It's not a typo, it's an adjective, not a noun. Backlight of the keyboard vs a backlit keyboard. 

Example: Your keyboard is backlit? Cool! vs Adjust the backlight brightness.

**Comment by agucova** [comment]
About the format, as I described, it was done with the default settings of `clang-format` on each C file.  See [here](https://www.kernel.org/doc/html/latest/process/clang-format.html)

File: README.md (hunk 1 above)
Comment (glats) [reply README.md]: aah brrp. looks good tho

**Pull request merged**